FWCore/Integration/plugins/AcquireIntESProducer.cc

```diff
@@ -144,6 +171,25 @@ namespace edmtest {
     return std::make_unique<ESTestDataB>(11);
   }
 
+  std::unique_ptr<AcquireIntESProducer::TestValue> AcquireIntESProducer::acquireUniquePtr(
+      ESTestRecordI const& record, edm::WaitingTaskWithArenaHolder holder) {
+    usleep(200000);
+    holder.doneWaiting(std::exception_ptr{});
```

Comment:
Do you have a specific reason to call `WaitingTaskWithArenaHolder::doneWaiting()` explicitly and in this position in the function?

Reply:
I deleted that line. It was harmless, but served no purpose. The destructor takes care of it.